controls/V-72841.rb

```diff
@@ -20,6 +20,12 @@
 uri: http://iase.disa.mil
 -----------------
 =end
+
+PG_VER = attribute(
+  'pg_version',
+  description: "The version of the PostgreSQL process which is being inspected (tested)",
+)
```

Comment:
You can use the base `postgres` resource to get the version number for the default.  The user can override it in the attributes file if needed or via the `.kitchen.yml` file.  We should keep the variable and reference the same name - i.e. either `pg_ver` or `pg_version` for clarity.
```
PG_VERSION = attribute(
  'pg_version',
  description: "The version of the PostgreSQL process which is being inspected (tested)",
  default: inspec.postgres.version
)
```